packages/react-spinner/src/components/Spinner/useSpinner.tsx

```diff
@@ -16,6 +16,29 @@ import { DefaultSvg } from './DefaultSvg';
 export const useSpinner_unstable = (props: SpinnerProps, ref: React.Ref<HTMLElement>): SpinnerState => {
   // Props
   const { appearance = 'primary', labelPosition = 'after', size = 'medium', status = 'active' } = props;
+  const baseId = useId('spinner');
+
+  const { tabIndex, ...rest } = props;
+  const nativeRoot = getNativeElementProps('div', { ref, ...rest }, ['size']);
+
+  const labelShorthand = resolveShorthand(props.label, {
+    defaultProps: {
+      id: baseId,
+    },
+    required: false,
+  });
+
+  const spinnerShortHand = resolveShorthand(props.spinner, {
+    required: true,
+    defaultProps: {
+      children: <DefaultSvg />,
+      tabIndex: tabIndex,
```

Comment:
_nit:_ You could just do `tabIndex` here

 ```suggestion
      tabIndex,
```